database/user.go

```diff
@@ -551,13 +551,13 @@ func (db *DB) UserSetStripeID(ctx context.Context, u *User, stripeID string) err
 }
 
 // UserSetTier sets the user's tier to the given value.
-func (db *DB) UserSetTier(ctx context.Context, u *User, t int) error {
+func (db *DB) UserSetTier(ctx context.Context, u *User, t int, upsert bool) error {
 	if t <= TierAnonymous || t >= TierMaxReserved {
```

Comment:
Follow Up: worth connecting my other comment here as well.

Reply:
I revisited the need for the upsert flag and removed it.